include/vcpkg/installedpaths.h

```diff
@@ -20,6 +20,7 @@ namespace vcpkg
         Path vcpkg_dir_info() const { return vcpkg_dir() / "info"; }
         Path vcpkg_dir_updates() const { return vcpkg_dir() / "updates"; }
         Path lockfile_path() const { return vcpkg_dir() / "vcpkg-lock.json"; }
+        Path stampfile_path() const { return vcpkg_dir() / "stampfile"; }
```

Comment:
Hmmm... this is almost certainly not the correct path; it probably should be part of the installed tree rather than the vcpkg_root.

In particular, the vcpkg_root may be readonly.

(The lockfile is OK because it's a read / lock operation applied to the file rather than a write)

Reply:
`vcpkg_dir()` is the `vcpkg` dir inside of the install folder.

Reply:
See https://github.com/microsoft/vcpkg-tool/blob/b3b6577286e8c7791aa572c204ca2699ddc0de21/src/vcpkg/vcpkgpaths.cpp#L415-L435

Reply:
Megaderp, thanks